jenkins/bokchoy_db_pull_request.py

```diff
@@ -0,0 +1,212 @@
+import sys
+import logging
+import os
+
+import click
+from github import Github
+
+logging.basicConfig()
+logger = logging.getLogger()
+logger.setLevel(logging.INFO)
+
+DB_CACHE_FILEPATH = 'common/test/db_cache'
+
+BOKCHOY_DB_FILES = [
+    'bok_choy_data_default.json',
+    'bok_choy_data_student_module_history.json',
+    'bok_choy_migrations_data_default.sql',
+    'bok_choy_migrations_data_student_module_history.sql',
+    'bok_choy_schema_default.sql',
+    'bok_choy_schema_student_module_history.sql',
+    'bok_choy_default_migrations.yaml',
+    'bok_choy_student_module_history_migrations.yaml'
+]
+
+
+def _authenticate_with_github(github_token):
+    """
+    Authenticate with Github using a token.
+    Returns a github instance.
+    """
+    try:
+        github_instance = Github(github_token)
+    except:
+        raise StandardError(
```

Comment:
nit: could you reword the error message to specifically point to the fact that the token is what we need to check